types/cote/index.d.ts

```diff
@@ -171,7 +174,7 @@ export class Subscriber extends Component {
      * @param type Type. May be wildcarded or namespaced like in EventEmitter2.
      * @param listener Callback. Returns nothing.
      */
-    on<T extends Event>(
+    on<T extends Event = GenericEvent>(
```

Comment:
I think this makes it too easy to forget a type argument. If you really want this to be unchecked you can write `send<any>`.